pkg/cmd/issue/create/create_test.go

```diff
@@ -437,6 +436,8 @@ func TestIssueCreate_recover(t *testing.T) {
 	_, err = tmpfile.Write(data)
 	assert.NoError(t, err)
 
+	tmpfile.Close()
```

Comment:
Could this be moved up to where tmpfile was initialized and converted to `defer tmpfile.Close()`?